include/BufferManager.h

```diff
@@ -32,20 +32,18 @@
 namespace lmms
 {
 
+class SampleFrame;
 
 class LMMS_EXPORT BufferManager
 {
 public:
 	static void init( fpp_t fpp );
-	static sampleFrame * acquire();
+	static SampleFrame * acquire();
```

Comment:
```suggestion
	static SampleFrame* acquire();
```

Reply:
This same * issue comes in below lines too. Hope you will clear up everything without me telling the exact lines.

Reply:
Hi @Rossmaxx, all these lines were touched by a simple find/replace action. So I'd rather not change them any more. Changing everything to `SampleFrame*` would touch 114 files again and still leave problems like whitespace after parentheses or pointers directly attached to the variable name in changed lines.

The focus of this PR is on enabling the simplification of some DSP code and I think this is already quite some improvement.

Reply:
Nvm leave it then.

Reply:
The lines are already touched, change them please.